MMA Removal Utility/README.md

```diff
@@ -0,0 +1,288 @@
+| extend  OSType = tostring(parse_json(Metadata_s).OSType)
+| project VMResourceID = ResourceId, OSType;
+let virtualMachinesExtensions = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType =~ "VMExtension" and Source_s =~ "AzTS_05_VMExtensionInventoryProcessor"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend VMResourceID = tolower(substring(ResourceId,0,indexof(ResourceId, '/', 0, -1, 9 )))
+| extend ExtensionType = tostring(parse_json(Metadata_s).ExtensionType)
+| project ResourceId, VMResourceID, ExtensionType;
+let virtualMachinesWithBothExtensions = virtualMachines
+| join kind=leftouter (virtualMachinesExtensions) on VMResourceID
+| summarize Extensions = make_list(ExtensionType) by VMResourceID, OSType
+|where (Extensions contains "MicrosoftMonitoringAgent" or Extensions contains "OmsAgentForLinux") and (Extensions contains "AzureMonitorWindowsAgent" or Extensions contains "AzureMonitorLinuxAgent");
+virtualMachinesWithBothExtensions
+```
+
+### **8. How I can get list of all the VMs which MMA agent present?**
+VMs and Extensions inventory is collected in Log Analytics workspace. To list all the VMs which have both MMA agent present, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType =~ "VirtualMachine"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend  OSType = tostring(parse_json(Metadata_s).OSType)
+| project VMResourceID = ResourceId, OSType;
+let virtualMachinesMMAExtensions = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType =~ "VMExtension" and Source_s =~ "AzTS_05_VMExtensionInventoryProcessor"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend VMResourceID = tolower(substring(ResourceId,0,indexof(ResourceId, '/', 0, -1, 9 )))
+| extend ExtensionType = tostring(parse_json(Metadata_s).ExtensionType)
+| where ExtensionType =~ "OmsAgentForLinux" or ExtensionType =~ "MicrosoftMonitoringAgent"
+| project ResourceId, VMResourceID, ExtensionType, ExtensionResourceId = ResourceId;
+let virtualMachinesWithMMAExtensions = virtualMachines
+| join kind=inner  (virtualMachinesMMAExtensions) on VMResourceID
+| project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
+virtualMachinesWithMMAExtensions
+```
+### **9. Installation command is failing with error message, 'The deployment MMARemovalenvironmentsetup-20233029T103026 failed with error(s). Showing 1 out of 1 error(s). Status Message:  (Code:BadRequest). What should I do now?**
+
+We have observed this intermittent issue with App service deployment, please re-run the installation command with same parameter values. Command should proceed without any error in next attempt. If this doesn't help, please reach out to support team or log an issue.
```

Comment:
Pls. mention the support team's DL: aztssup@microsoft.com

Reply:
Done